Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
```

Comment:
### [rubocop-ast](https://www.rubocop.org/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.16.0 → 1.21.0

[_[change-log](https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop/rubocop-ast/), [gem-diff](https://my.diffend.io/gems/rubocop-ast/1.16.0/1.21.0)_]





<details>
<summary>Commits</summary>

A change of **40** commits. See the full changes on [the compare page](https://github.com/rubocop/rubocop-ast/compare/v1.16.0...v1.21.0).

These are the first 10 commits:
- (47ec985) [Restore docs/antora.yml](https://github.com/rubocop/rubocop-ast/commit/47ec985726ae45ec3fe33822b8b7d6d3459f3598)
- (9a203f7) [Add Ruby 3.1 to the CI matrix](https://github.com/rubocop/rubocop-ast/commit/9a203f7d17300daca099377a41472006c52b36a8)
- (16bf27b) [Suppress RuboCop RSpec 2.9's warnings (#225)](https://github.com/rubocop/rubocop-ast/commit/16bf27b9bb8cdfddd1afdec6f5bb98bf6fa6a8d7)
- (f5a1548) [Use `actions/checkout` version 3](https://github.com/rubocop/rubocop-ast/commit/f5a15487dc84e4c36d976df529368484952e4bd6)
- (8bb6113) [Make `Node#condition?` aware of `case-match` node](https://github.com/rubocop/rubocop-ast/commit/8bb61137a38d19c5db12e8f3fb577c7540411c10)
- (732e796) [Update Changelog](https://github.com/rubocop/rubocop-ast/commit/732e796c7714a5e1f44d68e6454b20e4d63d58bd)
- (d860b92) [Cut 1.17.0](https://github.com/rubocop/rubocop-ast/commit/d860b92817ad1801aec2a12eaa0996007b948f72)
- (51f938a) [Restore docs/antora.yml](https://github.com/rubocop/rubocop-ast/commit/51f938a2ef77c4c0364d55e598a8fa1dad375740)
- (016a2a0) [Suppress RuboCop offenses (#229)](https://github.com/rubocop/rubocop-ast/commit/016a2a078d5e764ade1b39892c8a49b2964458b6)
- (5b53037) [Drop Ruby 2.5 support](https://github.com/rubocop/rubocop-ast/commit/5b53037d322275cdd5082e4164f976146cc6c014)


</details>